lib/backend/model/node.go

```diff
@@ -33,6 +33,8 @@ var (
 	typeHostIp        = rawIPType
 	matchHostMetadata = regexp.MustCompile(`^/?calico/v1/host/([^/]+)/metadata$`)
 	matchHostIp       = regexp.MustCompile(`^/?calico/v1/host/([^/]+)/bird_ip$`)
+	orchRefMatch      = regexp.MustCompile(`^/?/calico/v1/host/.*/orchestrator/(.*)`)
```

Comment:
Is there a reason you decided to have each ref as its own path, versus having a single per-node orchestratorrefs section?

e.g. `calico/v1/host/.*/orchrefs/` and plop the data in as a single json blob?

Reply:
Not particularly, I don't see a huge advantage one way or the other, but I'd be happy to move over to storing it as a json blob if that's what we'd prefer.

Reply:
I think that's what I'd prefer.

Reply:
@robbrockbank ?

Reply:
oh yes - use a single entry in etcd.  The more entries we add the more things we need to query to query a single node.